ddtrace/contrib/redis/asyncio_patch.py

```diff
@@ -0,0 +1,29 @@
+from ddtrace import config
+
+from ...internal.utils.formats import stringify_cache_args
+from ...pin import Pin
+from .util import _trace_redis_cmd
+from .util import _trace_redis_execute_pipeline
+
+
+#
+# tracing async functions
+#
+async def traced_async_execute_command(func, instance, args, kwargs):
```

Comment:
https://github.com/DataDog/dd-trace-py/blob/0d62b96977545584c3204cfad2cbebc9c959ec1d/ddtrace/contrib/trace_utils_async.py#L13-L20 should be used here to deal with the pin boilerplate

Reply:
@Kyle-Verhoog WDYT?